internal/testdata/schema_registry_cluster/read_clusters_private_1jnw8z.json

```diff
@@ -20,8 +20,13 @@
           "related": "https://api.confluent.cloud/org/v2/environments/env-1jnw8z",
           "resource_name": "crn://confluent.cloud/organization=1111aaaa-11aa-11aa-11aa-111111aaaaaa/environment=env-1jnw8z"
         },
-        "http_endpoint": "",
+        "http_endpoint": "https://psrc-y1111.us-west-2.aws.confluent.cloud",
         "private_http_endpoint": "https://lsrc.us-west-2.aws.private.stag.cpdev.cloud",
```

Comment:
I was thinking only one of http_endpoint, private_http_endpoint can be specified at the same time

Reply:
Oh, that's good to know. Thanks for clarifying!